ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml

```diff
@@ -87,9 +87,28 @@ spec:
             ;;
         esac
 
-        # `grep .` ensures, that pipe fail if jq returns no results
-        cat $PROJECT_DATA_PATH | jq '.container.isv_pid' | tr -d '"' | grep . | tr -d "\n" | tee $(results.isv_pid.path)
-        cat $PROJECT_DATA_PATH | jq '.container.repository_name' | tr -d '"' | grep . | tr -d "\n" | tee $(results.repo_name.path)
-        cat $PROJECT_DATA_PATH | jq '.org_id' | tr -d '"' |  tr -d '"' | grep . | tr -d "\n" | tee $(results.org_id.path)
-        cat $PROJECT_DATA_PATH | jq '.container.github_usernames[]' | tr '\n' ' ' | tr -d '"' | grep . | tr -d "\n" | tee $(results.github_usernames.path)
-        cat $PROJECT_DATA_PATH | jq '.operator_distribution' |  tr -d '"' | grep . | tr -d "\n" | tee $(results.operator_distribution.path)
+
+        get_from_cp () {
+          # Function to get the necessary fields from the CertProject
+          FIELD_NAME=$1
+          JQ_CODE=$2
+
+          echo "Retrieving $FIELD_NAME from the Certification Project"
+          # `grep .` ensures, that pipe fail if jq returns no results
+          cat $PROJECT_DATA_PATH | jq -e $JQ_CODE | tr -d '"' | grep . | tr -d "\n" || \
```

Comment:
You can probably simplify this command with `jq -r` which will strip the quotes.